status-im/status-legacy
# Move installations to status-go

Move installation management to status-go.

This commits moves all the management of installations to status-go, so that realm is not used anymore, we can't delete the schema as we need to pull them to migrate, but we delete all the records as soon as they are migrated.

### Testing
- Pairing (enabled/disabled devices)
-  Upgrade from previous version

For the upgrade from previous version, we want to make sure all the devices and names are there enabled after the installation.
To test:
1) Create a set of devices and pair them using the old version.
2) Update to this PR
3) Turn on airplane mode before login (we don't want to receive some messages, so we can make sure that the migration went ok)
4) Check that devices contains the same list, with the devices correctly named and enabled/disabled

status: ready
